pom.xml

```diff
@@ -597,6 +599,65 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
             </dependency>
           </dependencies>
         </plugin>
+        <plugin>
+          <!-- plugin to generate OpenAPI documentation for JAXRS REST Services -->
+          <groupId>io.swagger.core.v3</groupId>
+          <artifactId>swagger-maven-plugin</artifactId>
+          <version>${io.swagger.version}</version>
+          <dependencies>
+            <dependency>
+              <groupId>io.swagger.core.v3</groupId>
+              <artifactId>swagger-jaxrs2</artifactId>
+              <version>${io.swagger.version}</version>
+            </dependency>
+            <dependency>
+              <groupId>javax.ws.rs</groupId>
+              <artifactId>javax.ws.rs-api</artifactId>
+              <version>2.1.1</version>
+            </dependency>
+            <dependency>
+              <groupId>javax.servlet</groupId>
+              <artifactId>javax.servlet-api</artifactId>
+              <version>4.0.1</version>
+            </dependency>
+          </dependencies>
+          <executions>
+          <execution>
+            <phase>compile</phase>
+            <goals>
+              <goal>resolve</goal>
+            </goals>
+          </execution>
+        </executions>
+        </plugin>
+        <plugin>
+          <!-- plugin to generate OpenAPI documentation for Spring REST Services -->
+          <groupId>io.github.kbuntrock</groupId>
+          <artifactId>openapi-maven-plugin</artifactId>
```

Comment:
We need more official plugins please, unless it's impossible to do so.

Reply:
I did not find another working plugin for creating swagger documentation for Spring Services.
In addition, there is any plugin allowing to create swagger doc for both spring AND jaxrs Services in one time, so that I use the openapi-merger plugin, which is the only one I found.